api/src/BcGov.Malt.Web/Startup.cs

```diff
@@ -57,23 +60,35 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication(options =>
             {
-                options.DefaultAuthenticateScheme = Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
             }).AddJwtBearer(o =>
             {
+                /* TODO Add keycloak authentication params once its ready .The following may change in future
+                 * For now , Jwt:Authority = the url to your local keycloak relam , eg: Master or ISB
+                 *               Audience = the name of the client you create in keycloak relam ,e.g: maltd or demo-app 
+                 *   "Jwt":{
+                             "Authority": "http://localhost:8080/auth/realms/<Relam_Name>",
+                             "Audience": "maltd"
+                } */
                 o.Authority = Configuration["Jwt:Authority"];
                 o.Audience = Configuration["Jwt:Audience"];
-                o.Events = new Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerEvents()
+                o.Events = new JwtBearerEvents()
                 {
                     OnAuthenticationFailed = c =>
                     {
                         c.NoResult();
 
-                        c.Response.StatusCode = 500;
+                        c.Response.StatusCode = 401;
                         c.Response.ContentType = "text/plain";
-                      
-                            return c.Response.WriteAsync(c.Exception.ToString());
-                                              
+
+                        _log.Error(c.Exception, "Failed to authenticate. ");

```

Comment:
Extra space at the end of the log message and there should be no period. It is also recommended that log events should be phrases not sentences, see [Writing Log Events](https://github.com/serilog/serilog/wiki/Writing-Log-Events) for more information